docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,80 @@
-The system is smart enough not to rebuild applications that have already been built,
-so if you have multiple applications, only changed applications are rebuilt and redeployed.
+Run build hook
+: The build hook comprises one or more shell commands that you write to finish creating your production code base.
+It could be compiling Sass files, running a bundler, rearranging files on disk or compiling an application in a compiled language.The committed build hook runs in the build container. During this time, commands have write access to the file system, but services (including other applications in your cluster) aren't yet available.
 
-{{< /note >}}
+Freeze app container
+: Once the previous steps are completed, the file system gets freezed and produce a read-only container image, which is the final build artifact. An artifact is a custom-built snapshot of your application that's reliable and repeatable.
+The container configuration for both your application and its underlying services is exclusively based on your configuration files.
+A container has a 1:1 relationship with a Git commit.
+This means that builds are repeatable and that if the system detects that there aren't changes for a container,
+it reuses the existing container image skipping the build step and saving time.
 
-The live environment is composed of multiple containers&mdash;both for your applications and for the services it depends on.
-It also has a virtual network connecting them as well as a router to route incoming requests to the appropriate application.
 
-Based on your application type, the system selects one of the pre-built container images and runs the following:
+## The deploy
+The deploy process makes those containers live, replacing the previous versions, with virtually no interruption in service.
 
-1. First, any dependencies specified in the `.platform.app.yaml` file are installed.
-   Those include tools like Sass, webpack, and Drupal Console.
-2. Then, depending on the build flavor specified in the configuration file, a series of standard commands is run.
-   The default for PHP containers, for example, is running `composer install`.
 
-3. Finally, the build hook from the configuration file is run.
-   The build hook comprises one or more shell commands that you write to finish creating your production code base.
-   That could be compiling Sass files, running a bundler, rearranging files on disk,
-   compiling an application in a compiled language, or whatever else you want.
-   Note that, at this point, all you are able to access is the file system; there are no services or other databases available.
+### Deploy steps
 
-Once all of that is completed, we freeze the file system and produce a read-only container image.
-That image is the final build artifact:
-a reliable, repeatable snapshot of your application, built the way you want, with the environment you want.
+Hold requests
+: The system holds incoming requests so there is no service interruption.
```

Comment:
I feel like using the passive would be better than having "the system" doing things in some of these places"